docs/setup.rst

```diff
@@ -0,0 +1,28 @@
+Account setup
+=============
+
+Creating an account on StarkNet
+-------------------------------
+
+An account is needed to start interacting with StarkNet. There are a few ways of creating one:
+
+ - using DeployAccount transaction
+ - deploy through Cairo syscall (another account is needed)
```

Comment:
I think we should mention UDC here, even if it is not ready yet.

Reply:
I'd say it's a bit confusing to the users to read the guide for a library and see mentions to features that not exist and are not supported at all.